cmd/tailscale/cli/network-lock.go

```diff
@@ -369,6 +370,16 @@ func runNetworkLockRemove(ctx context.Context, args []string) error {
 				}
 			}
 		}
+	} else {
+		fmt.Printf(`Warning
+Removal of a signing key(s) without resigning nodes (--re-sign=false)
+will cause any nodes signed by the the given key(s) to be locked out
+of the Tailscale network. Proceed with caution.
+`)
+		if !prompt.YesNo("Are you sure you want to remove the signing key(s)?") {
```

Comment:
We might want to add ability to skip the prompt to support non-interactive use cases - either an envvar you could set, or perhaps only show the prompt if `isatty.IsTerminal` is true (we use it already in a couple of other places)